cmd/lakectl/cmd/cat_hook_output.go

```diff
@@ -17,14 +18,14 @@ var catHookOutputCmd = &cobra.Command{
 	ValidArgsFunction: ValidArgsRepository,
 	Run: func(cmd *cobra.Command, args []string) {
 		u := MustParseRepoURI("repository", args[0])
-		Fmt("Repository: %s\n", u.String())
+		fmt.Printf("Repository: %s\n", u)
```

Comment:
```suggestion
		fmt.Println("Repository: %s", u)
```

Reply:
kept the format - can switch to use Println and drop the format but it is the same